### ISSUE TITLE
.NET 4.6.1 support for JsonNetSerializer 6.4.2

### ISSUE THREAD

**Issue by code-chris** [elastic/elasticsearch-net#3752]
**NEST/Elasticsearch.Net version**: 6.4.2

**Elasticsearch version**: 6.4.2

**Description of the problem including expected versus actual behavior**:
Hey all,
we use version 6.4.2 of the NuGet packages "Elasticsearch.NET", "NEST" and "NEST.JsonNetSerializer" in a .NET Framework 4.6.1 project.

We currently want to convert our csproj-Files to the new 2017 format to prepare for targeting netstandard2.0 in the future (for .NET Core migration).

Unfortunately the NEST.JsonNetSerializer nuget package does not target net461 too, as the other two packages do in this version. This breaks our project:
```
Package NEST.JsonNetSerializer 6.4.2 is not compatible with net461 (.NETFramework,Version=v4.6.1). Package NEST.JsonNetSerializer 6.4.2 supports: netstandard2.0 (.NETStandard,Version=v2.0)
    One or more packages are incompatible with .NETFramework,Version=v4.6.1.
```
Also we cannot convert our whole project to netstandard2.0 right now. So this dependency is a blocker for us now...

Is it possible to add this additional target to the nuspec file? Or is there another possible workaround?

Thanks!

CC @aschiffmann

**Comment by Mpdreamz** [comment]
netstandard2.0 will resolve for `4.6.1` if you are on later SDK's 

https://docs.microsoft.com/en-us/dotnet/standard/net-standard

However we will add the explicit `net461` in the next release so you do not have to rely on a lot of additional system packages to make `net461` compatible with `netstandard 2.0`

**Comment by code-chris** [comment]
@Mpdreamz 
Yes, netstandard2.0 resolves to 4.6.1 thats not the problem. The problem is only nuget, which fails on install, cause the target is not set in nuspec.

Do you fix that in the 6.4.x release-line or only in later versions? We cannot update to a newer Elastic/NEST version right now...

**Comment by Mpdreamz** [comment]
It will be fixed in `6.8.1` whats preventing you to upgrade to `6.8.X` right now?

**Comment by Mpdreamz** [comment]
To be clear NEST/Elasticsearch.NET should work against Elasticsearch 6.4.2, the only parity lock is on the major version.

**Comment by code-chris** [comment]
Yes, if NEST 6.8 is fully backward compatible to 6.4.x Elasticsearch we can try to upgrade. We cannot upgrade Elasticsearch itself cause of proprietary plugins.

**Comment by skepok** [comment]
Hey guys,

I'm also having this issue - .net461 - can only install JsonNetSerializer v6.0.2 - all later versions produce same issue listed above. 

Any idea when we'll see 6.8.1 released?

Cheers!

Scott

**Comment by russcam** [comment]
This is now fixed in d60353f129c3570d892d354677b9ecbd237ca625 and will be in the next 6.x release, 6.8.1, going out now.
